src/esp/bindings/SensorBindings.cpp

```diff
@@ -93,11 +98,29 @@ void initSensorBindings(py::module& m) {
       .def_readwrite("clear_color", &CameraSensorSpec::clearColor);
 
   // ====CameraSensorSpec ====
-  py::class_<CameraSensorSpec, CameraSensorSpec::ptr, VisualSensorSpec,
-             SensorSpec>(m, "CameraSensorSpec", py::dynamic_attr())
+  py::class_<CameraSensorSpec, CameraSensorSpec::ptr, VisualSensorSpec>(
+      m, "CameraSensorSpec", py::dynamic_attr())
       .def(py::init(&CameraSensorSpec::create<>))
       .def_readwrite("ortho_scale", &CameraSensorSpec::orthoScale);
 
+  // ====FisheyeSensorSpec ====
+  py::class_<FisheyeSensorSpec, FisheyeSensorSpec::ptr, VisualSensorSpec>(
+      m, "FisheyeSensorSpec", py::dynamic_attr())
+      .def(py::init(&FisheyeSensorSpec::create<>))
+      .def_readwrite("focal_length", &FisheyeSensorSpec::focalLength)
+      .def_readwrite("principal_point_offset",
+                     &FisheyeSensorSpec::principalPointOffset)
+      .def_readwrite("cubemap_size", &FisheyeSensorSpec::cubemapSize)
```

Comment:
We need to leave a comment here for the cubemapSize. By default, if the user does not set it, the system will adopt `min(resolution[0], resolution[1])`. If this value is set by the user, it will override the default value.